python/doc/examples/reliability_sensitivity/reliability/plot_estimate_probability_form.py

```diff
@@ -122,26 +122,26 @@
 
 # %%
 # Reliability index with Breitung approximation
-sorm_result.getGeneralisedReliabilityIndexBreitung()
+print(sorm_result.getGeneralisedReliabilityIndexBreitung())
 
 # %%
 # ... with Hohenbichler approximation
-sorm_result.getGeneralisedReliabilityIndexHohenbichler()
+print(sorm_result.getGeneralisedReliabilityIndexHohenbichler())
 
 # %%
 # .. with Tvedt approximation
-sorm_result.getGeneralisedReliabilityIndexTvedt()
+print(sorm_result.getGeneralisedReliabilityIndexTvedt())
 
 # %%
 # SORM probability of the event with Breitung approximation
-sorm_result.getEventProbabilityBreitung()
+print(sorm_result.getEventProbabilityBreitung())
 
 # %%
 # ... with Hohenbichler approximation
-sorm_result.getEventProbabilityHohenbichler()
+print(sorm_result.getEventProbabilityHohenbichler())
 
 # %%
 # ... with Tvedt approximation
-sorm_result.getEventProbabilityTvedt()
+print(sorm_result.getEventProbabilityTvedt())
 
 plt.show()
```

Comment:
```suggestion
sorm_result.getEventProbabilityTvedt()

```
plt.show() is completely useless since we are using the openturns.viewer module. Actually, you could even remove the "from matplotlib import pylab as plt" line at the beginning of the script. I would propose it with GitHub's suggestion tool if I could, but GitHub won't let me since it is too far from your own changes.